.github/workflows/build-deploy-forms.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Deploy Formio forms
```

Comment:
I would preferer to make it explicit that we are deploying the Form.io server itself, for instance "Build and Deploy Formio Server"